composer.json

```
@@ -1,13 +1,13 @@
 {
-    "name": "rector/rector-laravel",
+    "name": "driftingly/rector-laravel",
     "type": "rector-extension",
     "license": "MIT",
     "description": "Rector upgrades rules for Laravel Framework",
     "require": {
         "php": ">=8.1"
     },
     "require-dev": {
-        "rector/rector-src": "dev-main",
+        "rector/rector": "^0.14",
```

you need to try use `dev-main` before next Rector release and ensure it keep working:

```suggestion
        "rector/rector": "dev-main",
```